backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -94,6 +94,20 @@
                 "required": True,
                 "array": False,
             },
+            {
+                "label": "creator",
+                "key": "creator",
+                "type": "text",
```

Comment:
rebuilt the whole script, so this will be obsolete - but included this fix.

Reply:
Okay,, so let me merge first so that you can overwrite it then.